nextflow_schema.json

```diff
@@ -173,27 +174,27 @@
                     "fa_icon": "far fa-circle"
                 },
                 "annotation_prodigal_closed": {
                     "type": "boolean",
                     "description": "Does not allow partial genes on contig edges.",
-                    "help_text": "Suppresses the partial genes from being on contig edge to result in closed ends. Should be activated for genomes where it is sure the first and last bases of the sequence(s) do not fall inside a gene. Run together with `-p normal` (former `-p single`) .\n\n> Modifies tool parameter(s):\n> - PRODIGAL: `--closed/-c`",
+                    "help_text": "Suppresses the partial genes from being on contig edge to result in closed ends. Should be activated for genomes where it is sure the first and last bases of the sequence(s) do not fall inside a gene. Run together with `-p normal` (former `-p single`) .\n\n> Modifies tool parameter(s):\n> - PRODIGAL: `-c`",
                     "fa_icon": "fas fa-circle"
                 },
                 "annotation_prodigal_transtable": {
                     "type": "integer",
                     "default": 11,
                     "description": "Specifies the translation table used for gene annotation.",
-                    "help_text": "Specifies which translation table should be used for seqeunce annotation. All possible genetic code translation tables can be found [here](https://en.wikipedia.org/wiki/List_of_genetic_codes). The default is set at 11, which is used for standard Bacteria/Archeae.\n\n> Modifies tool parameter(s):\n> - PRODIGAL: `--trans_table/-g`",
+                    "help_text": "Specifies which translation table should be used for seqeunce annotation. All possible genetic code translation tables can be found [here](https://en.wikipedia.org/wiki/List_of_genetic_codes). The default is set at 11, which is used for standard Bacteria/Archeae.\n\n> Modifies tool parameter(s):\n> - PRODIGAL: `-g`",
                     "fa_icon": "fas fa-border-all"
                 },
                 "annotation_prodigal_forcenonsd": {
                     "type": "boolean",
                     "description": "Forces PRODIGAL to scan for motifs.",
-                    "help_text": "Forces PRODIGAL to a full scan for motifs rather than activating the Shine-Dalgarno RBS finder, the default scanner for PRODIGAL to train for motifs.\n\n> Modifies tool parameter(s):\n> - PRODIGAL: `--force_nonsd/-n`",
+                    "help_text": "Forces PRODIGAL to a full scan for motifs rather than activating the Shine-Dalgarno RBS finder, the default scanner for PRODIGAL to train for motifs.\n\n> Modifies tool parameter(s):\n> - PRODIGAL: `-n`",
```

Comment:
Just out of interest: Is there a reason why only the short command should be shown? Some nf-core convention?

Reply:
On my phone so only replying: no there is no convention. But in this case there was inconsistency between the pipeline parameters, some with only short and some with both. When I checked the `-h` of the tool it only had the short ones so I selected them here for safety...